portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,7 +14,9 @@
 
 package com.google.sps.servlets;
 
-import java.util.stream.Collectors;
+import com.google.gson.*;
```

Comment:
Please order imports alphabetically

Reply:
done!

Reply:
Is this part of Google's style guides or generally accepted ones? I don't have access to go/codestyles to check

Reply:
This is a part of Google's style guides: https://google.github.io/styleguide/